### ISSUE TITLE
Table size, reversing direction, waves

### ISSUE THREAD

**Issue by khgrap** [markroland/sand-table-pattern-maker#23]
Mark - great software!  I downloaded and tweaked it to handle my table.  I wasn't sure where to put this, so I apologize if this isn't the appropriate forum or place.  Two quick suggestions and an idea, if you're open to it (unfortunately I don't know how to write code, otherwise I'd offer to help!)...
1. Allow the table size defined in env. to drive the pattern size parameters in the pattern files.
2. Have the ability to change the direction of certain shapes.  For example, Spiral path draws counter clockwise in-to-out by default.  Being able to change the direction (draw clockwise in-to-out) allows alternating patterns (clockwise -> counterclockwise) to redistribute the sand into the corners that eventually get thinned out.  I was able to do this by manually changing the theta calc (-i/i_max)...
3. Waves.  Just standard waves.  With a rectangular table like mine, being able to wipe/erase with a wave pattern instead of zig-zag would be great.  Just an idea.
Thanks again for sharing your work!
-KHG

**Comment by markroland** [comment]
Thank you for the interest and the feedback! I'll take this under consideration for the future. After several weeks of work on this project I'm taking a little bit of a break right now.

**Comment by voskaca** [comment]
Is there a way to change the size of the table?  I haven't been able to change it when I open it directly from the browser and I have not been able to install from the down load.

I am new to all of this and ALL seems to be over my head.

Any help would be great.

**Comment by markroland** [comment]
@voskaca Yes, you can edit the table.x.max and table.y.max values in https://github.com/markroland/sand-table-pattern-maker/blob/master/assets/js/env.js. The numbers are in millimeters. Let me know if you have any other questions.

**Comment by voskaca** [comment]
Thank you very much!
I am totally new to all of this and really appreciate the help.
